.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -31,12 +31,20 @@ Once all issues have passed acceptance testing and have been merged into `master
 @core_devs I've just created the `<major>.<minor>.x` release branch. Please be aware that any further changes intended for this release will have to be merged to `master` then backported. Thanks!
 ```
 - [ ] Build a beta named `<major>.<minor>.<patch>-beta.1` by pushing a git tag and when CI completes successfully notify the QA team that it's ready for release testing.
+- [ ] Announce the start of release testing on the [CHT forum](https://forum.communityhealthtoolkit.org/c/product/releases/26), under the "Product - Releases" category using this template:
+```
+*Release testing has started for {{version}} of {{product}}*
+
+To get a sneak peak at this upcoming release, you can install `<major>.<minor>.<patch>-beta.1` on your testing environment.
```

Comment:
Seems like a good opportunity to encourage testing of project configs as well, which could identify config related regressions not found part of our testing , like [this one](https://forum.communityhealthtoolkit.org/t/translations-not-working-when-using-repeat-groups-with-dynamic-repeat-count/1713), which lead to https://github.com/medic/cht-core/issues/7515.

Here's some suggested text to sample from:
```suggestion
To get a sneak peak at this upcoming release, you can install `<major>.<minor>.<patch>-beta.1` on your testing environment. We suggest you test your forms and workflows with this release candidate version and raise any issues that you experience. This helps to to discover any potential regressions that wouldn't otherwise be caught during release testing.
```